### ISSUE TITLE
Multiple scorers

### ISSUE THREAD

**Issue by spaszek** [databricks/spark-sklearn#72]
Newest SKLearn docs(http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html) say that:

> scoring: For evaluating multiple metrics, either give a list of (unique) strings or a dict with names as keys and callables as values.

However, `spark-sklearn` does not seem to accept `scoring` parameter in a form of list.

Is there any easy way to accomplish that(submitting few scorers, eg. `scoring = ['accuracy', 'f1', 'roc_auc', 'average_precision']`? Or is it perhaps in your roadmap? If not, how much time(approximately) would the implementation take for somebody new to the project(like me)?

**Comment by jkbradley** [comment]
Hi @spaszek thanks for reporting this potential improvement.  I'm afraid we have very limited bandwidth to work on spark-sklearn, though it will be good to know if others need this feature so that we can prioritize.

**Comment by spaszek** [comment]
@jkbradley Fortunately I have my own bandwidth to spare for a good cause. Could I implement this feature? :)

**Comment by ELind77** [comment]
I would also really like this feature.

**Comment by hadyan-tvlk** [comment]
@spaszek Is there any update for this? were you already implement this? Thanks

**Comment by spaszek** [comment]
@hadyan-tvlk Unfortunately no - I'm still waiting for @jkbradley to respond whether databricks even accepts other people PRs... ;P

**Comment by chomchomtron** [comment]
I would really like this feature, too. Is there any plan to include it soon?
